server/text_generation_server/layers/linear.py

```diff
@@ -1,12 +1,19 @@
 import torch
 from text_generation_server.utils.import_utils import SYSTEM
 from torch.nn import functional as F
+import os
 
 if SYSTEM == "rocm":
-    try:
-        from vllm import _custom_C
-    except Exception as e:
-        raise ImportError(f"Could not load `vllm._custom_C`. Full error: {e}")
+    ROCM_USE_SKINNY_GEMM = os.getenv("ROCM_USE_SKINNY_GEMM", "True").lower() in (
+        "true",
+        "1",
+    )
+
+    if ROCM_USE_SKINNY_GEMM:
+        try:
+            from vllm import _custom_C
+        except Exception as e:
+            raise ImportError(f"Could not load `vllm._custom_C`. Full error: {e}")
```

Comment:
```suggestion
            raise ImportError(f"Could not load `vllm._custom_C` for ROCm skinny gemm. Full error: {e}")
```

or something like that?

Reply:
Added